src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+// Select the logo element
+const logo = document.querySelector('.logo');
+
+const clickPromise = new Promise((resolve, reject) => {
+  const timeoutId = setTimeout(() => {
+    reject('Promise was rejected!');
+  }, 3000);
+
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+    clearTimeout(timeoutId);
+  });
+});
+
+const messageDiv = document.createElement('div');
+
+document.body.append(messageDiv);
+
+clickPromise
+  .then((message) => {
+    messageDiv.classList.add('message');
+    messageDiv.textContent = message;
+  })
+  .catch((errorMessage) => {
+    messageDiv.classList.add('message', 'error-message');
```

Comment:
The 'error-message' class is added to the div only in the `catch` block. If you want to display error messages differently than success messages, you could add a separate class for success messages in the `then` block, and style them differently in your CSS.